### PR TITLE
Add navigation to dashboard & resources to lovelace editor + icons

### LINKED ISSUE #5172: Add dashboards/resources to Lovelace menu in Configure UI mode
## The request

When entering Configure UI mode, it might be helpful to show the new Dashboard panel and Resources editor inside that Lovelace menu.

image

### PR DESCRIPTION
**Pull request by spacegaier**
## Proposed change

Navigation options + icons as proposed in the linked FR.

image

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [X] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes https://github.com/home-assistant/frontend/issues/5172
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [X] The code change is tested and works locally.
- [X] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by Misiu** [comment]
@spacegaier the first element on the list doesn't have an icon.
Maybe the `view-list` icon?
image

**Comment by spacegaier** [comment]
I know, because I did not find a god one on my first glance. Your proposal looks quit similar to the next two which feels confusing. I would prefer something more distinct like this:

image

File: src/panels/lovelace/hui-root.ts [hunk 1]
```diff
@@ -169,17 +172,50 @@ class HUIRoot extends LitElement {
...
+                        "ui.panel.lovelace.editor.menu.manage_dashboards"
+                      )}
+                    </mwc-list-item>
+                    <mwc-list-item
+                      graphic="icon"
+                      @request-selected="${this._handleManageResources}"
+                    >
+                      <ha-svg-icon
+                        slot="graphic"
+                        .path=${mdiViewDashboardOutline}
```
Comment (bramkragten) [inline src/panels/lovelace/hui-root.ts:205]: We should use the same icon for resources everywhere, we now use a different icon in the config panels.

Comment (spacegaier) [reply src/panels/lovelace/hui-root.ts:205]: For the "Manage dashboards" I took the icon from the config panels. The resource tab however does not have a separate icon so I had to choose one unless I missing a place where resources already get an icon?

Comment (bramkragten) [reply src/panels/lovelace/hui-root.ts:205]: https://github.com/home-assistant/frontend/blob/dev/src/panels/config/lovelace/ha-config-lovelace.ts#L19

It is used on mobile

**Review by bramkragten — APPROVED** [review]

**Pull request merged**